### PR TITLE
cloudanix

### PR DESCRIPTION
**Pull request by resmaa**
This PR documents the cloudanix pack, applying the Packs doc layout. The content is also edited, and @ajinkya1986 has added some information. 

@karl-cardenas-coding: The terraform 'type' may not be correct. Could you please check it? Is type needed?

Netlify doesn't seem to be running for this PR  to provide a preview.

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/913#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @resmaa. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by mpurusottamc** [comment]
@zulfilee Can you please review the PR?

File: content/docs/06-integrations/00-cloudanix.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-cloudanix.md:38]:
```suggestion
Cloudanix Helm Chart supports the following versions with Palette.
```

File: content/docs/06-integrations/00-cloudanix.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-cloudanix.md:28]:
```suggestion
Users can initiate Jira workflows and target specific workloads, excluding containers, pods, or nodes, right from the Cloudanix Dashboard. This Helm Chart installs three Cloudanix services to enable container security capabilities. The services are listed below:
```

File: content/docs/06-integrations/00-cloudanix.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-cloudanix.md:19]:
```suggestion
Cloudanix container security add-on detects threats and abnormal behaviors in Kubernetes clusters. The Cloudanix dashboard visualizes the following:
```

File: content/docs/06-integrations/00-cloudanix.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-cloudanix.md:23]:
```suggestion
* Mapping the threats to workloads in containers, pods, nodes, etc.
```

File: content/docs/06-integrations/00-cloudanix.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-cloudanix.md:26]:
```suggestion
* The user who initiated the threat, and much more.
```

File: content/docs/06-integrations/00-cloudanix.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-cloudanix.md:25]:
```suggestion
* The command that initiated or created the thread
```

File: content/docs/06-integrations/00-cloudanix.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-cloudanix.md:60]:
```suggestion
# Prerequisites
```

Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-cloudanix.md:60]: @resmaa prerequisites should be at the top. This is information the reader should be exposed to early on, so let's move it to the top.

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
Feedback and comments below.

**Comment by ajinkya1986** [comment]
@resmaa Can you address the PR comments

File: content/docs/06-integrations/00-cloudanix.md
Comment (ritawatson) [inline content/docs/06-integrations/00-cloudanix.md:39]:
```suggestion
* Kubernetes 1.15.x and higher
```

File: content/docs/06-integrations/00-cloudanix.md
Comment (ritawatson) [inline content/docs/06-integrations/00-cloudanix.md:38]:
```suggestion
* Memory: 25 MiB
```

File: content/docs/06-integrations/00-cloudanix.md
Comment (ritawatson) [inline content/docs/06-integrations/00-cloudanix.md:44]: This line can be deleted.

File: content/docs/06-integrations/00-cloudanix.md
Comment (ritawatson) [inline content/docs/06-integrations/00-cloudanix.md:61]: Delete lines 58-60. We're no longer listing versions like this.

File: content/docs/06-integrations/00-cloudanix.md
Comment (ritawatson) [inline content/docs/06-integrations/00-cloudanix.md:37]:
```suggestion
* CPUs: 1
```

File: content/docs/06-integrations/00-cloudanix.md
Comment (ritawatson) [inline content/docs/06-integrations/00-cloudanix.md:4]:
```suggestion
metaDescription: 'Helm chart pack that provides security for Cloudanix containers in Palette'
```

File: content/docs/06-integrations/00-cloudanix.md
Comment (ritawatson) [inline content/docs/06-integrations/00-cloudanix.md:19]:
```suggestion
The Cloudanix pack is an add-on security pack that provides a dashboard to help you detect threats and abnormal behavior in your Kubernetes clusters. The Cloudanix dashboard displays the following:
```

File: content/docs/06-integrations/00-cloudanix.md
Comment (ritawatson) [inline content/docs/06-integrations/00-cloudanix.md:24]:
```suggestion
* Events associated with a threat
```

File: content/docs/06-integrations/00-cloudanix.md
Comment (ritawatson) [inline content/docs/06-integrations/00-cloudanix.md:23]:
```suggestion
* A mapping of threats to workloads in containers, pods, and nodes.
```

File: content/docs/06-integrations/00-cloudanix.md
Comment (ritawatson) [inline content/docs/06-integrations/00-cloudanix.md:25]:
```suggestion
* The command that caused the threat to occur.
```

File: content/docs/06-integrations/00-cloudanix.md
Comment (ritawatson) [inline content/docs/06-integrations/00-cloudanix.md:26]:
```suggestion
* The user who initiated the activity that caused the threat.
```

File: content/docs/06-integrations/00-cloudanix.md
Comment (ritawatson) [inline content/docs/06-integrations/00-cloudanix.md:28]:
```suggestion
Users can start Jira workflows and target specific workloads, excluding containers, pods, or nodes, from the Cloudanix Dashboard. The Cloudanix Helm Chart installs three Cloudanix services to enable container security capabilities:
```

File: content/docs/06-integrations/00-cloudanix.md
Comment (ritawatson) [inline content/docs/06-integrations/00-cloudanix.md:34]: Add boldface to these.

**Review by ritawatson — COMMENTED** [review]
I did a quick copy-edit. When you've added these comments, I can pull the branch and apply the pack layout to it.

**Comment by ajinkya1986** [comment]
Hi @ritawatson. I have made the changes and added examples for threat and unusual behavior. I have sent you the file on slack. Can you please review.

**Comment by spectro-prow** [comment]
@resmaa: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | e60692ec2795b8d4cfc94210f20a4893afd5288b | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/913/librarium_build/1638921420392108032) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by spectro-prow** [comment]
@resmaa: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 88805d7991c4e140cd7c3c64aaddda9c928f1019 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/913/librarium_build/1638922049952944128) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by spectro-prow** [comment]
@resmaa: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | e47cc4f86b3dd7b84d12d7917df66a3bd5834a9e | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/913/librarium_build/1638924568846405632) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/06-integrations/00-cloudanix.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-cloudanix.md:134]:
```suggestion
  type = "helm"
```

File: content/docs/06-integrations/00-cloudanix.md
Comment (karl-cardenas-coding) [inline content/docs/06-integrations/00-cloudanix.md:24]:
```suggestion
- Detecting files added or modified in sensitive directories.
```

**Review by karl-cardenas-coding — APPROVED** [review]
👍🏻

**Pull request merged**
